tests/utils/index.js

```diff
@@ -614,7 +614,7 @@ const revertDb = async (except, ignoreRefresh) => { //NOSONAR
     watcher?.cancel();
     await commonElements.closeReloadModal(true);
   } else if (needsRefresh) {
-    await watcher && watcher.promise; // NOSONAR
+    watcher && await watcher.promise;
```

Comment:
Did sonar actually catch this bug?

Reply:
No, it complained about not using `?` shortcut.